accel-pppd/ctrl/sstp/sstp.c

```diff
@@ -2049,44 +2088,6 @@ static int sstp_recv(struct triton_md_handler_t *h)
 	return 1;
 }
 
-static int sstp_write(struct triton_md_handler_t *h)
```

Comment:
there's no need to move code block and lose git blame, instead you can define forward prototype `static int sstp_write(struct triton_md_handler_t *h);` near existing `static int sstp_read_deferred(struct sstp_conn_t *conn);`

Reply:
Good point. Will update. Thank you.